pkg/sql/sqlbase/computed_exprs.go

```diff
@@ -148,24 +166,28 @@ func ProcessComputedColumns(
 	semaCtx := tree.MakeSemaContext(false)
 	semaCtx.IVarContainer = iv
 
-	computedExprs := make([]tree.TypedExpr, 0, len(cols))
-	for i, col := range computedCols {
-		expr, _, _, err := ResolveNames(exprs[i],
+	compExprIdx := 0
+	for _, col := range cols {
+		if !col.IsComputed() {
+			computedExprs = append(computedExprs, tree.DNull)
```

Comment:
do we have to add the null expression?

Reply:
Yes, that's part of this function's documentation. It parallels MakeDefaultExprs.